### ISSUE TITLE
Optionally switch aggregation: group by plugin first, then module/filename

### ISSUE THREAD

**Issue by hossam-magdy** [toomuchdesign/babel-timing#17]
## 🚀 Feature Proposal

It would be quite useful to show the statistics/timings grouped by babel-plugins first, and THEN module/filename.

## Motivation

The current implementation of this tool helps in spotting **which module/file in the codebase is the bottleneck** in the compilation time.

But what if this tool used in an existing and stable codebase at which compilation pipeline includes plugins that do unnecessary or unoptimized processing (which is likely). And there might be an alternative plugin already available.

In this case, it would be very useful to spot **which plugin (pluginAlias) is the bottleneck** in the compilation time (takes the most time to process all/any files it does).

## Example

The 1st/main table would be:
https://raw.githubusercontent.com/toomuchdesign/babel-timing/694cd496d7bc542d94d364328a2cac40a277fd80/screenshot-02.png
And when navigating INTO any pluginAlias, this table is shown:
https://raw.githubusercontent.com/toomuchdesign/babel-timing/694cd496d7bc542d94d364328a2cac40a277fd80/screenshot-01.png

So, basically, it is just showing the same tables but reversed in order and for sure reconsidering the calculations.

As an initial guess how a consumer would configure this, is a **boolean** value options.aggregateByPlugin: **true/false** which would be by default false so that the existing behaviour remains the same.

**Comment by toomuchdesign** [comment]
Solved by #24.
